src/main/java/com/gluonhq/substrate/target/AbstractTargetConfiguration.java

```diff
@@ -127,15 +125,15 @@ public boolean link(ProcessPaths paths, ProjectConfiguration projectConfiguratio
                     +gvmPath.toString());
         }
         ProcessBuilder linkBuilder = new ProcessBuilder("gcc");
-        Path linux = gvmPath.resolve(appName);
+        Path linuxPath = gvmPath.resolve(appName);
```

Comment:
That name is wrong anyhow, as it is nothing linux specific. I think it should be changed to "appPath" or so?